### DISCUSSION TITLE
Add multi hashtags support on  twitter route

### DISCUSSION THREAD

**Discussion by comeUpWithItLater** [DIYgod/RSSHub#12770] [Ideas]
Is it possible to multi hashtags support to  https://docs.rsshub.app/en/social-media.html#twitter-keyword ?  so that only tweets contain these hashtags will be included.  for example, when i pass '#AIJobs, #remoteJobs' to the route params. Only tweets contain both  '#AIJobs', '#remoteJobs' should be included. somewhat like  ![](https://brand24.com/blog/wp-content/uploads/2022/06/how-to-search-multiple-hashtags-on-twitter-min.jpg)

**Comment by TonyRL** [comment]
https://docs.rsshub.app/en/parameter.html#filtering
